wof/examples/production_configs/wofpy.conf

```diff
@@ -1,5 +1,5 @@
 # description of the script purpose
-description "uWSGI server instance configured to serve timeseries"
+description "uWSGI server instance configured to serve WOFpy"
```

Comment:
I've generalized the setup so that it would work for multiple dao's or wofpy app such as having `timeseries` and `measurement` etc.